misc/python/materialize/mzcompose/composition.py

```diff
@@ -645,6 +645,17 @@ def run(
             check=check,
         )
 
+    def run_testdrive(
```

Comment:
How about the `def testdrive`, You need to also put `capture_stderr=True` there as well.

Reply:
Yeah, that's done in the other PR.

Reply:
Oh, I understand, you mean https://github.com/MaterializeInc/materialize/pull/24542/files#diff-5615adde56537a8e03acb4effc63219d8bb5e71e3f9c8083a8bbe74946fbb28fR1069. Need to check if and how that overlaps with my method.
Thanks for bringing this up!

Reply:
That's used when running a string. I should maybe rename my method then to `run_testdrive_file`. `capture_stderr=True` is already set there.